cli/tests/integration/lsp_tests.rs

```diff
@@ -1420,6 +1421,61 @@ fn lsp_format_mbc() {
   shutdown(&mut client);
 }
 
+#[test]
+fn lsp_format_exclude() {
+  let temp_dir = TempDir::new().expect("could not create temp dir");
```

Comment:
This is failing on mac because the canonicalized version of TempDir is different. I think we need to change everything related to the config file and lsp to just use specifiers as it's not possible to canonicalize some documents that are open, but not saved to the file system. I am looking into this now.

Reply:
Should be fixed on the Mac CI now in ee84932e4b47267de18c8d53b22f1a4f2f253699 by using module specifiers in the config file.